tensorflow/python/compiler/tensorrt/utils.py

```diff
@@ -86,3 +88,97 @@ def is_experimental_feature_activated(feature_name):
   return (feature_name
           in os.environ.get("TF_TRT_EXPERIMENTAL_FEATURES",
                             default="").split(","))
+
+
+def get_nodes_in_engine(graphdef, node_name):
+  ops_in_engine = collections.defaultdict(int)
+  for func in graphdef.library.function:
+    if f"{node_name}_native_segment" == func.signature.name:
+      node_count = len(func.node_def)
+      for node in func.node_def:
+        ops_in_engine[node.op] += 1
+      break
+  return node_count, ops_in_engine
+
+
+def export_graph_to_dot(graphdef, dot_output_filename):
```

Comment:
Similar to the above, please add document to the function.

Reply:
Fixed